### PR TITLE
fix: prevent modal calls on selection

### PR DESCRIPTION
**Pull request by pyphilia**
Selecting items trigger lots of renders.
- Moved down DragSelection (so some duplication happens). Some manual adjustement is necessary because of margin and padding.
-  do not fetch items for modal if they are not opened.

### CODE REVIEW

File: src/components/item/FolderContent.tsx [hunk 1]
```diff
@@ -49,6 +50,10 @@ const Content = ({ item, searchText, items, sortBy }: Props) => {
   const { itemTypes } = useFilterItemsContext();
   const { selectedIds, clearSelection, toggleSelection } =
     useSelectionContext();
+  const theme = useTheme();
+  const DragSelection = useDragSelection({
+    adjustments: { marginTop: 70, marginLeft: theme.spacing(3) },
```
Comment (spaenleh) [inline src/components/item/FolderContent.tsx:55]: Where are these defined ? can we use the same constants ?

File: src/components/main/list/useDragSelection.tsx [hunk 2]
```diff
@@ -0,0 +1,76 @@
...
+    onSelectionEnd: () => {},
+    selectionProps: {
+      style: {
+        // adjustement
+        // https://github.com/AirLabsTeam/react-drag-to-select/issues/30
+        ...adjustments,
+        border: `2px dashed ${PRIMARY_COLOR}`,
+        borderRadius: 4,
+        backgroundColor: 'lightblue',
+        opacity: 0.5,
```
Comment (spaenleh) [inline src/components/main/list/useDragSelection.tsx:69]: Should we also increase the zIndex so it appears on top of the drawer ?

**Review by spaenleh — APPROVED** [review]

File: src/components/pages/RecycledItemsScreen.tsx [hunk 3]
```diff
@@ -51,70 +53,77 @@ const RecycledItemsScreenContent = ({
...
-            </Stack>
-          )}
+            )}
+          </Stack>
+          {
+            // render the filtered data and when it is empty display that nothing matches the search
+            filteredData?.length ? (
+              filteredData.map((item) => (
+                <ItemCard
+                  // todo: should not be able to click on the card
```
Comment (ReidyT) [inline src/components/pages/RecycledItemsScreen.tsx:102]: Still something to do ?

**Review by ReidyT — APPROVED** [review]
LGTM 👑

File: src/components/item/FolderContent.tsx (hunk 1 above)
Comment (pyphilia) [reply src/components/item/FolderContent.tsx:55]: These depends on the layout.. so it's kind of unstable, but it depends on many params unfortunately =/

**Pull request merged**